<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TYSS" name="testscripts.TC_33_MobileNumberWithAlphanumericValue" tests="2" failures="0" timestamp="2021-12-16T17:03:12 IST" time="6.193" errors="1">
  <testcase name="MobileNumberWithAlphanumericValueTest" time="6.193" classname="testscripts.TC_33_MobileNumberWithAlphanumericValue"/>
  <system-out/>
  <testcase name="closeDB" time="0.000" classname="testscripts.TC_33_MobileNumberWithAlphanumericValue">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.cams.GenericLibrary.DatabaseUtility.closeDb(DatabaseUtility.java:28)
at com.cams.GenericLibrary.BaseClass.closeDB(BaseClass.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- closeDB -->
  <system-out/>
</testsuite> <!-- testscripts.TC_33_MobileNumberWithAlphanumericValue -->
